translations/messages.da.yaml

```diff
@@ -807,7 +807,7 @@ sync_status:
     ok: '✓'
     button: 'Kør!'
     not_started: 'Job sat i kø'
-    running: 'I gang...'
+    running: 'I gang..'
```

Comment:
This actually should be
"i gang..."

So it becomes "Synkronisering i gang..."

<img width="229" height="53" alt="Screenshot 2025-08-04 at 11 29 19" src="https://github.com/user-attachments/assets/bde2590a-e7c7-4c6f-a535-1ccded6f8c4c" />